pkgs/development/tools/language-servers/fish-lsp/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchYarnDeps,
+  fixup-yarn-lock,
+  fetchFromGitHub,
+  yarn,
+  fish,
+  nodejs,
+}: let
+  version = "1.0.6";
```

Comment:
There's no tag `1.0.6`.

Reply:
Oh no ! 

I will ask the author of `fish-lsp` to tag the latest commit with `v1.0.6`, if the author don't want to tag, I will put like I did originally the commit id.

So this PR can wait maybe a day or two before the author respond.

Reply:
> It looks like there's a 1.0.6 and a 1.0.7 tag now (https://github.com/ndonfris/fish-lsp/tags). Is there anything I can do to help move this forward?

Thanks for you message, It's been almost a week I was testing in local my package do verify if every was working that I forgot I need one day to push my change.
I update my package for 1.0.7